docker-compose.yml

```diff
@@ -3,24 +3,13 @@ services:
   editor:
     build:
       context: .
+      args:
+        REACT_APP_SPOOF_SINOPIA_SERVER: 'false'
+        REACT_APP_SINOPIA_SERVER_URL: http://platform:8080
     ports:
       - 8000:8000
     depends_on:
-      - pipeline
-  pipeline:
```

Comment:
Current work does not require the pipeline or elasticsearch. Only requires Trellis. So removing these from the docker-compose config for now.